h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -103,6 +109,71 @@ public void initMetalearnerParams(Metalearner.Algorithm algo) {
     public Key<Frame>[] _base_model_predictions_keys; 
   }
 
+  public static class StackedEnsembleMRScorer extends MRTask<StackedEnsembleMRScorer> {
```

Comment:
name is a bit misleading: strictly speaking, it doesn't "score" the model, it's preparing the level one frame that will be used to score the metalearner: what about `BaseModelsMRScorer`?